prog/redeliver_github_failures.rb

```diff
@@ -2,12 +2,9 @@
 
 class Prog::RedeliverGithubFailures < Prog::Base
   label def wait
-    current_frame = strand.stack.first
-    last_check_time = Time.parse(current_frame["last_check_at"])
+    last_check_time = Time.parse(strand.stack.first["last_check_at"])
```

Comment:
Could use `frame` here, though it's a bit less efficient.

```suggestion
    last_check_time = Time.parse(frame["last_check_at"])
```